setup.py

```diff
@@ -22,11 +22,11 @@
     python_requires=">=3.6",
     install_requires=[
         "click<8.0.0",
-        "click-shell",
+        "click-shell~=2.0",
         "packaging",
```

Comment:
It seems like we are always are bumping the major and our use of packaging is very limited so decided not to bother pinning.